drivers/modem/quectel-bg9x.c

```diff
@@ -894,17 +900,19 @@ static void pin_init(void)
 	k_sleep(K_MSEC(250));
 #endif
 
-	/* NOTE: Per the BG95 document, the Reset pin is internally connected to the
-	 * Power key pin.
-	 */
+	modem_pin_write(&mctx, MDM_RESET, 1);
```

Comment:
Is this sequence for 'any quectel bg9x' ? If not, then use a build define to distinquish between the 'stock' quectel initialization routine and this one.